<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderCommentWidget</class>
 <widget class="QWidget" name="OrderCommentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>347</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FinanMemberWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#FinanMemberWidget{background-color: rgb(255, 255, 255);border-radius:8px;}</string>
  </property>
  <widget class="QPushButton" name="buttonOK">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>72</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>32</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2, 164, 253, 1), stop:1 rgba(31, 139, 237, 1));border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
			QPushButton:hover,pressed{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(40, 178, 253, 1),stop:1 rgba(64, 156, 240, 1));
		  border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
QPushButton:disabled{ background-color:rgba(200,200,200,1);border-radius:4px;};</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>400</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>18</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>18</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>请对本次服务进行评价</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonGrade1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="EntCenterWidget.qrc">
     <normaloff>:/EntCenterWidget/image/faspace.png</normaloff>:/EntCenterWidget/image/faspace.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonGrade2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="EntCenterWidget.qrc">
     <normaloff>:/EntCenterWidget/image/faspace.png</normaloff>:/EntCenterWidget/image/faspace.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonGrade3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="EntCenterWidget.qrc">
     <normaloff>:/EntCenterWidget/image/faspace.png</normaloff>:/EntCenterWidget/image/faspace.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonGrade5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="EntCenterWidget.qrc">
     <normaloff>:/EntCenterWidget/image/faspace.png</normaloff>:/EntCenterWidget/image/faspace.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonGrade4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="EntCenterWidget.qrc">
     <normaloff>:/EntCenterWidget/image/faspace.png</normaloff>:/EntCenterWidget/image/faspace.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelGrade">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>400</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>一般</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDelay">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>问题处理不及时</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonService">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>工作人员态度不好</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonProfessional">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>工作人员不够专业</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="editContent">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>320</width>
     <height>71</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>填写更多的内容，以帮助我们更好的为您服务</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>72</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>32</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>暂不评价</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EntCenterWidget.qrc"/>
 </resources>
 <connections/>
</ui>
